src/containers/pages/FocusInvestigation/jurisdiction/index.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { ObjectList } from '../../../../helpers/cbv';
+import { Message } from '../../../../store/tests/ducks/messages';
+
+interface Props {
+  messages: Message[];
+}
+
+const options = {
+  dispatchPropName: 'loadMessages',
+  listPropName: 'messages',
+};
+
+/** this is our dumb component */
+const FIJurisdiction = (props: Props) => {
```

Comment:
@p-netm @cKellyDesign this is sample usage of `ObjectList`. How do you feel about this?  What can be better?

Reply:
I think the reducerName would have to be placed somewhere different, at the moment, i feel like its more natural to have the dumb component either statically define the reducerName or pass an option to the HOC that tells it what reducer slice to register or use

Reply:
You're right.  This part I had not worked on yet.  It also ties in nicely with what we talked about the other day i.e. generic ducks

Reply:
> i feel like its more natural to have the dumb component either statically define the reducerName or pass an option to the HOC that tells it what reducer slice to register or use

Agreed!